src/main/java/com/gregtechceu/gtceu/integration/jei/GTJEIPlugin.java

```diff
@@ -69,7 +67,7 @@ public void registerCategories(@NotNull IRecipeCategoryRegistration registry) {
         if (ConfigHolder.INSTANCE.machines.doBedrockOres)
             registry.addRecipeCategories(new GTBedrockOreInfoCategory(jeiHelpers));
         for (GTRecipeCategory category : GTRegistries.RECIPE_CATEGORIES) {
-            if (category.shouldRegisterDisplays()) {
+            if (GTCEu.isDev() || category.shouldRegisterDisplays()) {
```

Comment:
shouldRegisterDisplays already checks for dev env - please remove otherwise this bypasses the furnace recipe blacklist